README

```diff
@@ -1,11 +1,11 @@
 Pants Docker
-Version 0.1
+Version 1.0
```

Comment:
I figure it's good enough for us to publish a new major version and anything still using the old line can pin to v0.1, but let me know if you're not comfortable starting 1.x and we can re-evaluate.

Reply:
oh we'd also need to update the version in the python distribution target

Reply:
I think the version in the python distribution target is done in the `BUILD` file below @compyman, does it look correct to you?

https://github.com/compyman/pants-docker/pull/17/files#diff-0a65a761614b486c966b9093381282a0f143212ec96812a18beb4d47bf3732f6R14-R17